### PR TITLE
show success message after restore

### PR DESCRIPTION
**Pull request by MaximilianMeister**
this is just nicer than saying nothing at all

### CODE REVIEW

**Comment by tboerger** [comment]
Do we display successful messages now? In general we have removed that.

**Comment by MaximilianMeister** [comment]
> Do we display successful messages now? In general we have removed that.

`/installer/installer`, `/installer/upgrade/restore`, `/installer/upgrade/finishing` come to my mind, have a success message

in some critical steps it makes just sense to report about success IMO

**Comment by vuntz** [comment]
Can you rebase?

**Comment by vuntz** [comment]
> in some critical steps it makes just sense to report about success IMO

I agree, but better to check with @kwwii

**Comment by MaximilianMeister** [comment]
@vuntz @kwii approved the changes

**Comment by vuntz** [comment]
+1

**Comment by rsalevsky** [comment]
+1

File: crowbar_framework/config/locales/crowbar/en.yml [hunk 1]
```diff
@@ -708,7 +708,7 @@ en:
       crowbar: 'Crowbar'
       openstack: 'OpenStack'
       restore: 'Restore'
-      restore_warning: 'By pressing OK, you will start the restoration process for the system. Are you sure?'
+      restore_warning: 'Before pressing OK, it is advised to download the Backup(s) as they will not be restored. By pressing OK, you will start the restoration process for the system. Are you sure?'
```
Comment (tboerger) [inline crowbar_framework/config/locales/crowbar/en.yml]: By pressing OK, ....... By pressing OK?

Comment (MaximilianMeister) [inline crowbar_framework/config/locales/crowbar/en.yml]: no, it is `Before pressing OK .... By pressing OK` that's a difference

Comment (tboerger) [inline crowbar_framework/config/locales/crowbar/en.yml]: That still sounds strange to me.

Comment (MaximilianMeister) [inline crowbar_framework/config/locales/crowbar/en.yml]: > That still sounds strange to me.

any better idea?

**Comment by vuntz** [comment]
Can you backport to 3.0?

**Pull request merged**